aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/UpdateHandler.java

```diff
@@ -49,6 +45,14 @@ public UpdateHandler(TagHelper tagHelper) {
         final ResourceModel model = request.getDesiredResourceState();
         // if failed, return previous resource state
         final ResourceModel previousModel = request.getPreviousResourceState();
+
+        if(model == null){
+            ProgressEvent.<ResourceModel, CallbackContext>builder()
```

Comment:
why would model be null, and why this case is returned as success?

Reply:
Here model states the current update event process, if it has invalid data or no update state, then by defauult we are returning the previous model state which will have event status details with the message

Reply:
if you want to check invalid data, wouldn't you need a full validation test?

have you seen cases or known from CFN that it's possible to call updateHandler with no update state? 

I just want to make sure every piece of code is there have some functionality.

Reply:
and where would the message go? i.e. who sees the message, in what format?
I dont' recall there is specific place in the code to handle the returned message.

Reply:
We have validated the changes with unit test cases & contract test.
In both test , the process was able to handle the both success & failure events  and its passed the tests.

The contract test input sample files are there for updateHandler, so when the process is invoke those details will invoked.

Reply:
I still don't understand who process the message, and where does it go, who's able to see it

Reply:
if you feel it is not okay to keep there i will comment this.

Reply:
I don't know, if you think this is a necessary step, you can add it there, I don't have the full context, just based on recollection. 
My question is still you put the message in the response, where does the message go, who's able to see it in what format? and your replay did not answer this question.

Reply:
i reverted back to the original because it is not affecting the original implementation.